src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1107,6 +1107,14 @@ describe('ReactDOMComponent', function() {
         'Warning: This browser doesn\'t support the `onScroll` event'
       );
     });
+
+    it('should not warn when server-side rendering `onScroll`', function() {
+      spyOn(console, 'error');
+      ReactDOMServer.renderToString(React.createElement('div', {
```

Comment:
I'm surprised this would warn normally since we're in jest and have a DOM, even for server rendering. I would expect isEventSupported to return true (that's why we explicitly mock it in the prior test) and in turn, we wouldn't warn. But I did run locally and saw the warning… oh well.